lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -88,6 +88,7 @@
   <a href="./doesnotexist" target="_blank">internal link is ok</a>
 </template>
 
+<script>window.Error = function(error) { this.stack = 'stacktrace'; };</script>
```

Comment:
will you add a comment about why we're doing this and maybe referencing #1194 for future reference?